### PR TITLE
added further usage and troubleshooting steps

### PR DESCRIPTION
**Pull request by AmyD97**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -77,10 +103,44 @@ If there is anything wrong with this repository (documentation needed / broken t
...
+- The time that the vagrant takes to build the box is heavily dependent on your machine's hardware. It can definitely take much longer than expected.
+- Make sure your machine has at least 4 MB available of RAM (memory) to provide to the Box. If your machine has less than that, modify the vagrant file vb.memory value to be less than 50% of your machine's RAM.
+
+### Virtual Box display seems pixelated or shows fuzz or static
+
+For windows, follow these [instructions](https://superuser.com/questions/894328/virtualbox-guests-are-upsampled-blurry-due-to-dpi-scaling-from-windows-host-a). You may need to click aditionally on "Change high DPI settings". Then, shut down the VM as per the VM Usage Guide and restart VirtualBox.
+
+## Q & A
+
+### How does the repository connect to the VM?
```
Comment (Cybower) [inline README.md]: Everyone no longer needs to do this. I added the other repositories as submodules to this repository.

File: README.md [hunk 2]
```diff
@@ -77,10 +103,44 @@ If there is anything wrong with this repository (documentation needed / broken t
...
+- If an error occurs in GitHub or in your code editor such as "*Sorry, you're not able to edit this repository directly -- You need to fork it and propose changes from there instead*", please ask access to the repository by sending a direct message via Discord to the repository owner. Once access has been granted by the owner, an email will be sent with an invitation to email linked to your GitHub account. Please click the "Accept Invitation" button on the email. This should resolve the issue.
+
+### Vagrant is not working
+
+- If you just installed Vagrant, the device on which it was installed will need to be restarted before the software is usable.
+- Otherwise, please repair vagrant by opening the installer and clicking "Next" > "Repair".
+
+### Vagrant up is taking forever
+
+- The time that the vagrant takes to build the box is heavily dependent on your machine's hardware. It can definitely take much longer than expected.
```
Comment (Cybower) [inline README.md]: The box is "built" already, its more dependent on your Storage (i.e. HD vs SSD r/w) and internet download speeds.  The Box file that gets download with `vagrant up` is approx 10 Gb

File: README.md [hunk 3]
```diff
@@ -37,13 +39,37 @@ If there is anything wrong with this repository (documentation needed / broken t
...
    1. uname: `vagrant`
    2. password: `vagrant`
 
+
 **Note** You can alter the vb.memory value in the Vagrantfile based on how much RAM your personal machine has. I wouldn't give it more than %50 of what your system max is
 
+## VM Usage Guide
+1. To start the machine run the following command from within the directory of this cloned repository (i.e. `/eagle-development-environment/`):
+   `vagrant up`
+2. To open Mission  Planner:
```
Comment (Cybower) [inline README.md]: Might want to add separate sections detailing guides for each of the tools we're using (Mission Planner, MAVProxy, ArduPilot, etc.)

**Review by Cybower — COMMENTED** [review]
Really liked the FAQ Section! Definitely want to keep an up to date headache tracker to see whats causing the most trouble and hopefully answer the questions :) 

FYSA I added a markdown linter action (think gitlab pipeline), so we'll need to clean up the formatting a little bit for that to pass. In a recent push into main, I included my .vscode folder with a few settings files that make it substantially easier to see the best practices with an extension called Prettier.

File: README.md (hunk 3 above)
Comment (AmyD97) [reply README.md]: I will add as I keep compiling info

File: README.md (hunk 2 above)
Comment (AmyD97) [reply README.md]: changed to load

File: README.md (hunk 1 above)
Comment (AmyD97) [reply README.md]: Added C2 Tag

**Comment by AmyD97** [comment]
@Cybower please review before I can merge

**Review by Cybower — APPROVED** [review]

**Pull request merged**